vendor/engines/saas/app/jobs/saas/plans_destroy_job.rb

```diff
@@ -0,0 +1,11 @@
+module Saas
+  class PlansDestroyJob < ActiveJob::Base
+
+    def perform(params)
+      if params[:event] == 'plan_canceled'
+        params['url'] = "/settings/#{params[:repo_owner]}/canceled"
```

Comment:
This should be `canceling`; `canceled` is for hits on a board that has already been canceled.